### PR TITLE
font-latin-modern: use https in URL

### PR DESCRIPTION
**Pull request by i0ntempest**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-fonts/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Review by bevanjkay — CHANGES_REQUESTED** [review]
@i0ntempest Can you please also add a `desc` stanza?

**Comment by i0ntempest** [comment]
> @i0ntempest Can you please also add a `desc` stanza?

Sure but how should I describe fonts (groups)?

**Comment by i0ntempest** [comment]
> @i0ntempest Can you please also add a `desc` stanza?

Descriptions added, tho not sure if they are any good.

File: Casks/font-latin-modern.rb [hunk 1]
```diff
@@ -2,9 +2,10 @@
   version "2.004"
   sha256 "5b0236051d3728be6616f1b274e3b910473875b5a3ef9382f0ef00384ddb161b"
 
-  url "http://www.gust.org.pl/projects/e-foundry/latin-modern/download/lm#{version}otf.zip"
+  url "https://www.gust.org.pl/projects/e-foundry/latin-modern/download/lm#{version}otf.zip"
   name "Latin Modern"
-  homepage "http://www.gust.org.pl/projects/e-foundry/latin-modern"
+  desc "The Latin Modern font family, derived from Computer Modern fonts"
```
Comment (miccal) [inline Casks/font-latin-modern.rb:7]:
```suggestion
  desc "Latin Modern font family, derived from Computer Modern fonts"
```

File: Casks/font-latin-modern.rb [hunk 2]
```diff
@@ -2,9 +2,10 @@
...
+  homepage "https://www.gust.org.pl/projects/e-foundry/latin-modern"
```
Comment (miccal) [inline Casks/font-latin-modern.rb:9]:
```suggestion

  livecheck do
    url "https://www.gust.org.pl/projects/e-foundry/latin-modern/download"
    regex(/lm(\d+(?:\.\d+)+)otf\.zip/i)
  end

```

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**